File: Projet java/brazil burger/src/entity/Paiement.java
package entity;

public class Paiement {

    private String mode;
    private double montant;

    public Paiement(String mode, double montant) {
        this.mode = mode;
        this.montant = montant;
    }

    @Override
    public String toString() {
        return "Paiement " + mode + " : " + montant + " FCFA";
    }
}

File: Prjojet symphony/src/Entity/Paiement.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

#[ORM\Entity]
class Paiement
{
    #[ORM\Id]
    #[ORM\GeneratedValue]
    #[ORM\Column]
    private ?int $id = null;

    #[ORM\Column(type: 'datetime')]
    private ?\DateTimeInterface $datePaiement = null;

    #[ORM\Column]
    private ?float $montant = null;

    #[ORM\Column(length: 10)]
    private ?string $mode = null;

    #[ORM\OneToOne]
    private ?Commande $commande = null;

    public function getId(): ?int { return $this->id; }
}
